.github/workflows/ci.yml

```diff
@@ -38,7 +38,8 @@ jobs:
       - uses: julia-actions/julia-processcoverage@v1
       - uses: codecov/codecov-action@v5
         with:
-          file: lcov.info
+          files: lcov.info
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Assuming there is an organization secret already of this name. If this fail, then an org admin needs to go add that.